README.md

```diff
@@ -1163,6 +1163,8 @@ Converts the case of a string or of all strings in an array to lowercase.
 
 #### `empty`
 
+**Deprecated. Puppet 5.5.0 has introduced a built in `empty` function, which will take precedence over the stdlib function. See [Puppet 5.5.0 Release Notes](https://puppet.com/docs/puppet/5.5/release_notes.html#new-features).**
```

Comment:
The functions should only be deprecated for users of puppet >=5.5.0, but not for users of puppet <5.5.0. We can remove them from stdlib entirely when we drop compatibility for puppet <6.0.0 from stdlib in 2050 or whatever :P.